src/js/components/Box/Box.js

```diff
@@ -53,7 +53,7 @@ class Box extends Component {
             contents.push(
               <StyledBoxGap
                 // eslint-disable-next-line react/no-array-index-key
```

Comment:
Can you remove this comment now? If not, then I don't think the change really does anything. The key value never shows up in the DOM.

Reply:
@ericsoderberghp no, I've tried :(
It is still valid because we are still using the array index, even after concatenating a string to it.